schemas/com.amazon.aws.lambda/java-context/jsonschema/1-0-0

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
```

Comment:
Can you add a description line with link back to the Lambda documentation? See e.g. https://github.com/snowplow/iglu-central/blob/master/schemas/com.segment/alias/jsonschema/1-0-0#L3